avaframe/com1DFAPy/com1DFA.py

```diff
@@ -829,9 +863,26 @@ def initializeResistance(cfg, dem, logName, resLine):
         log.info('Resistance area: %s' % (resistanceArea))
         mask = prepareArea(resLine, dem)
         cResRaster = 0.5 * d * cw / (sres*sres) * mask
+        areaInfo['resistance'] = 'Yes'
+    elif ('res' in logName) and resLine:
+        # resistanceArea = os.path.splitext(os.path.basename(resFile))[0]
+        resistanceArea = resLine['Name']
+        log.info('Resistance area: %s' % (resistanceArea))
+        mask = prepareArea(resLine, dem)
+        cResRaster = 0.5 * d * cw / (sres*sres) * mask
+        areaInfo['resistance'] = 'Yes'
+    elif ('all' in logName) and resLine:
+        # resistanceArea = os.path.splitext(os.path.basename(resFile))[0]
+        resistanceArea = resLine['Name']
+        log.info('Resistance area: %s' % (resistanceArea))
+        mask = prepareArea(resLine, dem)
+        cResRaster = 0.5 * d * cw / (sres*sres) * mask
+        areaInfo['resistance'] = 'Yes'
```

Comment:
same here, we could just use a flag that we define line 216 and co

Reply:
done but also changed